primitives/runtime/src/lib.rs

```diff
@@ -285,6 +285,28 @@ pub fn storage_value_key(pallet_prefix: &str, value_name: &str) -> StorageKey {
 	StorageKey(final_key)
 }
 
+/// Can be use to access the runtime storage key of a `StorageDoubleMap`.
```

Comment:
Iiuc it isn't about `StorageDoubleMap`, but about `StorageMap`

Reply:
True. Done.